src/domain/services/filter-avaliable-events.ts

```diff
@@ -0,0 +1,15 @@
+import { Event } from '../entities';
+
+export function filterAvaliableEvents(eventList: Event[]): Event[] {
+  const currentDate = new Date();
+
+  const avaliableEvents: Event[] = [];
+
+  eventList.map((event) => {
+    if (event.eventDate >= currentDate) {
+      avaliableEvents.push(event);
+    }
+  });
```

Comment:
Maybe here we can use the filter() method which is specific for arrays? This way if nothing is found it returns an empty array by default.

```suggestion
  return eventList.filter((event) => {
    return event.eventDate >= currentDate;
  });
```

Reply:
Got it, but the  **return** in ` return event.eventDate >= currentDate;` is not necessary